torch_xla/torch_xla.py

```diff
@@ -59,24 +61,31 @@ def sync():
   xm.mark_step()
 
 
-@contextlib.contextmanager
-def step():
+def step(f: Optional[Callable] = None, *, eager_mode=False):
```

Comment:
within a `step` user would not want to set `eager` to True right?

Reply:
Haha I added this as an example usage of an argument. I'll go ahead and remove it -- you're right that it doesn't logically make sense here.